tasks/main.yml

```diff
@@ -10,17 +10,19 @@
     - "setup-{{ ansible_distribution|lower }}.yml"
     - "setup-{{ ansible_os_family|lower }}.yml"
 
-- name: Enable WireGuard kernel module
-  modprobe:
-    name: wireguard
-    state: present
-  register: wireguard_module_enabled
-  until:  wireguard_module_enabled is succeeded
-  retries: 10
-  delay: 10
-  failed_when: wireguard_module_enabled is failure
-  tags:
-    - wg-install
+- block:
+    - name: Enable WireGuard kernel module
+      modprobe:
+        name: wireguard
+        state: present
+      register: wireguard_module_enabled
+      until:  wireguard_module_enabled is succeeded
+      retries: 10
+      delay: 10
+      failed_when: wireguard_module_enabled is failure
+      tags:
+        - wg-install
+  when: not ansible_os_family == 'Darwin'
```

Comment:
Why did you wrap this one task in a block?

Reply:
Because on macOS there's no kernel module to enable. You need to manually run the Go binary (or use the official App).

Reply:
My question is just about the syntax :) The `when` condition can also directly be attached to this one `modprobe` task.

Reply:
AFAIR that was a suggestion by me. But you are right the `block` isn't needed. The code looked a little bit different at the beginning. That's an oversight of me.

Reply:
Oups, sorry. Should I open another PR to fix this?

Reply:
I guess you can do that. That is probably cleaner.